vagrant/scripts/configure_ntp.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+NTP_SERVER=$1
+
+sudo systemctl disable --now chronyd
+sudo yum install -y ntp
+# delete all server entries
+sudo sed -i -e "/^server /d" /etc/ntp.conf
+# Append ntp server address 
+sudo sed -i -e "$ a server $NTP_SERVER iburst" /etc/ntp.conf
+sudo systemctl restart ntpd.service
+sudo systemctl enable ntpd.service
+
+sleep 30
+
+until ntpstat;
+do
+  systemctl restart ntpd.service
```

Comment:
Instead of restarting ntpd in a loop, we should do something like:

`ntpdate -qu 10.73.10.1`

Reply:
this command would be run inside the loop?